docs/Using-Virtual-Environment.md

```diff
@@ -0,0 +1,52 @@
+# Using Virtual Environment
+
+## What is a Virtual Environment?
+A Virtual Environment is a self contained directory tree that contains a Python installation 
+for a particular version of Python, plus a number of additional packages.  
+
+## Why should I use a Virtual Environment?
+A Virtual Environment keeps all dependencies for the project separate from dependencies 
```

Comment:
We need to clarify what we mean by a project. As for Unity devs they will have Unity projects. We should also clarify that we expect them to have a single venv for all of ml-agents, aside from one per Unity "project".

Reply:
I think it's clear from the "What is a Virtual Environment?" answer that the term refers to a Python project. To be specific I have changed "dependencies for the project" to "dependencies for the Python project". 

Why do you expect users to maintain a single venv for all of ml-agents? To me that defeats the purpose of using virtual environments.